torch/csrc/distributed/rpc/rref_context.cpp

```diff
@@ -505,6 +508,16 @@ void RRefContext::delPendingUser(const ForkId& forkId) {
   deletedState.reset(); // Decrease refcount.
 }
 
+void RRefContext::addConfirmedUser(
```

Comment:
should we also call this function in delPendingUser to make it clear that confirmedUsers_ are updated in two different places? If doing that, will need to drop the lock in this function, and add a comment saying that the call site needs to acquire the lock.

Reply:
Sure, doing this.